t/t7524-serialized-status.sh

```diff
@@ -333,4 +333,59 @@ test_expect_success 'renames' '
 	test_i18ncmp output.1 output.2
 '
 
+test_expect_success 'hint message when cached with u=complete' '
+	git init hint &&
+	echo xxx >hint/xxx &&
+	git -C hint add xxx &&
+	git -C hint commit -m xxx &&
```

Comment:
Wouldn't `test_commit -C hint xxx` be more succinct than those three lines? (Of course, the file is then called `xxx.t`, but it would be easy to adapt the test case to that, no?)